Decisions.TruCap/Api/DocumentMonitorResponse.cs

```diff
@@ -67,15 +66,8 @@ public class DocumentMonitorResponse
         
         public static DocumentMonitorResponse[]? JsonDeserialize(string json)
         {
-            try
-            {
-                DocumentMonitorResponse[]? text = JsonConvert.DeserializeObject<DocumentMonitorResponse[]>(json);
-                return text;
-            }
-            catch (Exception e)
-            {
-                throw new BusinessRuleException(e.Message);
-            }
+            DocumentMonitorResponse[]? text = JsonConvert.DeserializeObject<DocumentMonitorResponse[]>(json);
+            return text;
```

Comment:
Same comment as above. Wondering if we should return an empty array if the nullable variable doesn't have a value. Also, text is a bad variable name since we're not returning text.